tests/ingresstests/l7_graph_test.go

```diff
@@ -124,6 +124,8 @@ func TestMain(m *testing.M) {
 	integrationtest.KubeClient = KubeClient
 	integrationtest.AddDefaultIngressClass()
 	ctrl.SetSEGroupCloudNameFromNSAnnotations()
+	integrationtest.AddDefaultNamespace()
```

Comment:
It is better to reuse the existing AddNamespace function. It already has the same functionality.

Reply:
It is the same for the UT w/o Infrasetting per NS